### PR TITLE
bluepill 3.0.0

### PR DESCRIPTION
**Pull request by fxcoudert**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by bayareabear** [comment]
Thanks!

**Comment by bayareabear** [comment]
@fxcoudert, The build failed. Could you please add the following to the build:
  xcodebuild \
    -project Bluepill-cli/Bluepill-cli.xcodeproj \
    -scheme BluepillLib \
    -configuration Debug \
    -derivedDataPath "build/" | tee result.txt

Before:
  xcodebuild \
    -workspace Bluepill.xcworkspace \
    -scheme bluepill \
    -configuration Release \
    -derivedDataPath "build/" | tee result.txt 

The latter depends on the first one. This is not ideal. I am going to fix it in the next pull request. Thanks!

**Comment by fxcoudert** [comment]
```
==> xcodebuild -project Bluepill-cli/Bluepill-cli.xcodeproj -scheme BluepillLib  -configuration Release SYMROOT=../
Build settings from command line:
    SYMROOT = ../

xcodebuild: error: The project named "Bluepill-cli" does not contain a scheme named "BluepillLib ". The "-list" option can be used to find the names of the schemes in the project.
```

@bayareabear

**Comment by bayareabear** [comment]
Release 3.0.1 that fixed the dependency issue. You don't need to add BluepillLib build any more:
xcodebuild
-project Bluepill-cli/Bluepill-cli.xcodeproj
-scheme BluepillLib
-configuration Debug
-derivedDataPath "build/" | tee result.txt

**Comment by oliverhu** [comment]
The error msg in travis doesn't seem to be reproducible locally :(

File: Formula/bluepill.rb [hunk 1]
```diff
@@ -14,8 +14,8 @@ class Bluepill < Formula
   depends_on :xcode => ["9.0", :build]
 
   def install
-    xcodebuild "-workspace", "Bluepill.xcworkspace",
```
Comment (oliverhu) [inline Formula/bluepill.rb]: Please revert this portion of change when you build with bluepill 3.0.1.

**Review by oliverhu — CHANGES_REQUESTED** [review]

**Comment by oliverhu** [comment]
two more things.. 1) please revert the change in `def install ` 2) please wait for a Bluepill 3.0.2 release, the 3.0.1 release might not work for homebrew distribution

**Comment by fxcoudert** [comment]
@bayareabear @oliverhu I am closing this pull request. Please open a new one when there is a version of bluepill that works fine in Homebrew.

**Pull request closed without merging**